src/main/java/com/wojtek/parkingmeter/helpers/HasStartedJSON.java

```diff
@@ -1,4 +1,20 @@
 package com.wojtek.parkingmeter.helpers;
 
+import com.wojtek.parkingmeter.helpers.enums.HasStartedEnum;
+
 public class HasStartedJSON {
+
+    HasStartedEnum hasStarted;
```

Comment:
dlaczego dostęp pakietowy?

Reply:
Faktycznie, zgapiłem. Skoro mam settery i gettery to dałbym private.